### PR TITLE
[Backend] Introduce Executor class

### PR DESCRIPTION
**Pull request by YongseopKim**
Let's introduce Executor class.

ONE-vscode-DCO-1.0-Signed-off-by: Yongseop Kim <<yons.kim@samsung.com>>

### CODE REVIEW

**Comment by jyoungyun** [comment]
I will wait for #490, #488 PR to be merged.
After merged these PRs, please uncomment your codes.

**Review by hyunsik-yoon — APPROVED** [review]
For the reason at https://github.com/Samsung/ONE-vscode/pull/488#issuecomment-1108033950

**Review by jyoungyun — APPROVED** [review]
LGTM

@YongseopKim intends to uncomment and use it whenever the user needs. It will be reflected as it is.

**Pull request merged**
